### PR TITLE
fix: updated settings page design

### LINKED ISSUE #14321: [Bug]: Design on settings tab is different for API query, DB query and JS editor and needs to be consistent for all these pages
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

Settings tab desgin is different on API/DB query and JS objects page and need to be aligned with each other

### Steps To Reproduce

image
image
image

### Public Sample App

_No response_

### Version

Cloud

### PR DESCRIPTION
**Pull request by AmanAgarwal041**
## Description

- Changed the component in `SwitchControl` from `Toggle` to `Switch`
- Updated the settings json for every plugin to have `subtitle` property instead of `info` property as per designs

Fixes #14321

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

> Please describe the tests that you ran to verify your changes. Provide instructions, so we can reproduce.
> Please also list any relevant details for your test configuration.

- Test A
- Test B

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by ayushpahwa — DISMISSED** [review]

File: app/client/src/constants/AppsmithActionConstants/formConfig/ApiSettingsConfig.ts [hunk 1]
```diff
@@ -6,44 +6,40 @@ export default [
...
+        subtitle:
           "Encode query params for all APIs. Also encode form body when Content-Type header is set to x-www-form-encoded",
       },
       {
         label: "Smart JSON Substitution",
         configProperty: "actionConfiguration.pluginSpecifiedTemplates[0].value",
-        controlType: "CHECKBOX",
-        info:
+        controlType: "SWITCH",
+        // encrypted: true,
```
Comment (sneha122) [inline app/client/src/constants/AppsmithActionConstants/formConfig/ApiSettingsConfig.ts:30]: can we please remove the commented code if not required?

**Review by sneha122 — DISMISSED** [review]

File: app/client/src/constants/AppsmithActionConstants/formConfig/ApiSettingsConfig.ts (hunk 1 above)
Comment (AmanAgarwal041) [reply app/client/src/constants/AppsmithActionConstants/formConfig/ApiSettingsConfig.ts]: Done

**Review by sneha122 — APPROVED** [review]

**Pull request merged**